modules/ts/src/ts_gtest.cpp

```diff
@@ -9505,6 +9505,7 @@ void FilePath::Normalize() {
 # include <io.h>
 # include <sys/stat.h>
 # include <map>  // Used in ThreadLocal.
+# include <crtdbg.h>
```

Comment:
Use the same preprocessor guard as constants below: `#ifdef _MSC_VER` != `#if GTEST_OS_WINDOWS` (to avoid break MINGW builds)

BTW, It is 3rdparty file. It is better to avoid changes here.

Reply:
Could you, please, elaborate a bit?
Why do you consider this files as 3rdparty?

Reply:
Declaration and usage should have the same preprocessor checks to avoid issues.

Reply:
Could you, please, point to the gtest original of this file?
We do use gtest yet we do not have any similar patches.